### PR TITLE
Build: Add CI release scripts (fxies #6884)

### PR DESCRIPTION
**Pull request by nzakas**
**What issue does this pull request address?**

#6884 

**What changes did you make? (Give an overview)**

* Added `ci-release` script
* ~~Added `gh-release` script~~

**Is there anything you'd like reviewers to focus on?**

Ensure changes in `Makefile.js` make sense.

### CODE REVIEW

**Comment by ilyavolodin** [comment]
LGTM, but waiting another day for others to look

**Comment by platinumazure** [comment]
LGTM.

**Comment by nzakas** [comment]
These particular changes may not be necessary - let's hold off so I can verify.

File: Makefile.js [hunk 1]
```diff
@@ -650,7 +651,7 @@ target.gensite = function(prereleaseVersion) {
                 }
                 const added = versions.added[baseName];
 
-                if (!versions.removed[baseName] && !test("-f", sourcePath)) {
+                if (!versions.removed[baseName] && !fs.existsSync(sourcePath)) {
```
Comment (ilyavolodin) [inline Makefile.js]: Slightly different logic here, right? Before it was checking if it exists and if it's a file, not it just checks if it exists.

Comment (nzakas) [inline Makefile.js]: Oops, not sure how this got in. Removing.

**Comment by ilyavolodin** [comment]
LGTM

**Pull request merged**